charts/fleet/templates/deployment_gitjob.yaml

```diff
@@ -59,4 +59,5 @@ spec:
         runAsNonRoot: true
         runAsUser: 1000
         runAsGroup: 1000
+        readOnlyRootFilesystem: true
```

Comment:
Do we want to also, as done in other deployments:
* run containers unprivileged?
* prevent privilege escalation?
* drop all POSIX capabilities?
If not, why not?